charts/wallabag/Chart.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v2
+appVersion: 2.4.1
+description: a self hostable application for saving web pages, freely.
```

```suggestion
description: A self hostable application for saving web pages, freely.
```

Mainly so that it renders nicely in the rendered README